CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - Fix: bug having entities without attibutes AND matching registrations made the attr-less entity not be returned by GET /v2/entities (#3176)
```

Comment:
PR shows a renaming:

scripts/managedb/upgrade-0.14.1/swap_coords.js → scripts/managedb/swap_coords.js

which I understand is incorrect and should be rolled back

(I'm adding this comment in another file, as github doesn't allow me to provide a comment on the renamed file itself)

Reply:
Perhaps caused by update with master, I will check it.

Reply:
Fixed in 77002fc